logistics_project/apps/malawi/warehouse/report_views/consumption_profiles.py

```diff
@@ -87,18 +87,34 @@ def get_consumption_profile_table(self, request, supply_point, extra_table_param
         table.update(extra_table_params)
         return table
 
+    def get_reporting_location_type(self, request, supply_point):
+        if request.base_level_is_hsa:
+            if is_country(supply_point):
+                location_type = "national"
+            elif is_district(supply_point):
+                location_type = "district"
+            elif is_facility(reporting_supply_point):
+                location_type = "facility"
+            else:
+                raise config.BaseLevel.InvalidReportingSupplyPointException(supply_point.code)
+        elif request.base_level_is_facility:
+            if is_country(supply_point):
+                location_type = "national"
+            elif is_district(supply_point):
+                location_type = "district"
+            else:
+                raise config.BaseLevel.InvalidReportingSupplyPointException(supply_point.code)
+        else:
+            raise config.BaseLevel.InvalidBaseLevelException(supply_point.code)
+
+        return location_type
```

Comment:
nit but i think you could remove the variable and just make everything a `return`